app/src/main/java/mainburg/planetenweg/LocationMarker.java

```diff
@@ -136,14 +164,16 @@ public void onLocationChanged(Location location) {
                 .title(activity.getResources().getString(R.string.your_location)));
         position.setIcon(BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_GREEN));
 
-        map.moveCamera(CameraUpdateFactory.newLatLngZoom(loc, map.getCameraPosition().zoom));
+        if (trackLocation) {
```

Comment:
We  should probably allow to lock the view on this location by clicking on it, but I couldn't be bothered to read the API docs for the `GoogleMap`.